hooks/boot/13-tuning.rb

```diff
@@ -1,7 +1,7 @@
 if has_custom_fact?('tuning')
   # TODO: automatically get the tuning sizes - standard doesn't exist and only
   # loads base. The rest maps to config/foreman.hiera/tuning/sizes/$size.yaml
-  TUNING_SIZES = ['default'] + ['medium', 'large', 'extra-large', 'extra-extra-large']
+  TUNING_SIZES = ['default'] + ['medium', 'large', 'extra-large', 'extra-extra-large', 'development'].freeze
```

Comment:
I wonder if it should freeze the whole:
```suggestion
  TUNING_SIZES = (['default'] + ['medium', 'large', 'extra-large', 'extra-extra-large', 'development']).freeze
```

Or it could be a single array.
```suggestion
  TUNING_SIZES = ['default', 'medium', 'large', 'extra-large', 'extra-extra-large', 'development'].freeze
```